conda_libmamba_solver/solver.py

```diff
@@ -321,128 +303,104 @@ def _solving_loop(
                 {
                     name: record.to_match_spec()
                     for name, record in in_state.installed.items()
-                    # TODO: These conditions might not be needed here
                     if not record.is_unmanageable
-                    # or name not in in_state.history
-                    # or name not in in_state.requested
-                    # or name not in in_state.pinned
```

Comment:
Huh, old stuff?

Reply:
Mostly some notes I kept around in case the refactor back then introduced solving issues (this would have been closer to what `classic` was doing). But we didn't get reports about this so I chose to clean up and forget.